source/extensions/filters/common/expr/context.h

```diff
@@ -54,6 +54,7 @@ constexpr absl::string_view Connection = "connection";
 constexpr absl::string_view MTLS = "mtls";
 constexpr absl::string_view RequestedServerName = "requested_server_name";
 constexpr absl::string_view TLSVersion = "tls_version";
+constexpr absl::string_view ConnectionTerminationDetails = "connection_termination_details";
```

Comment:
This should be `termination_details`. Also, please update RBAC docs.

Reply:
Thanks..FIxed